Formula/geos.rb

```diff
@@ -12,7 +12,6 @@ class Geos < Formula
   end
 
   option :cxx11
-  option "with-php", "Build the PHP extension"
```

Comment:
The PHP option needs to be removed because it is completely broken. Beginning with 3.6.0, the PHP bindings were moved from this library, the core library, into a separate project.